### DISCUSSION TITLE
Using the keyring on Linux (Freedesktop Secret service)

### DISCUSSION THREAD

**Discussion by VannTen** [ankitpokhrel/jira-cli#566] [Show and tell]
Hi.

I've faced some issues on setting up `jira-cli` using KeepassXC as my secret service (via the freedesktop secret service dbus api), so why not share that and avoid others the struggle.

1. You need to have a `login` collection exposed through the Dbus Secret Service API, as go-keyring only search in that one.
2. To add your token: `secret-tool store --label 'Jira Personal Access Token' service jira-cli username <your login>`
(That last step took me a long time, I kept trying to use : `secret-tool store --label 'Jira Personal Access Token' jira-cli <your login>`

Hope that will be useful to others :)

(Maybe that could be added to https://github.com/ankitpokhrel/jira-cli/discussions/356 ? )

**Comment by ankitpokhrel** [comment]
Added to #356, thank you @VannTen!
